slither/tools/read_storage/read_storage.py

```diff
@@ -42,18 +44,36 @@ class SlitherReadStorageException(Exception):
     pass
 
 
+class RpcInfo:
+    def __init__(self, rpc_url: str, block: Union[int, str] = "latest") -> None:
+        assert isinstance(block, int) or block.isnumeric() or block == "latest"
```

Comment:
What is `block.isnumeric()`

Reply:
Since `block` can be a string, I figured I'd play it safe in case `block == "12345678"` or something like that. Not necessary?

Reply:
I'm not sure. It's probably fine. I think there's also arguments besides "latest" like "finalized"

Reply:
Ahh that's a good point, I'll make sure "latest", "earliest", "pending", "safe" and "finalized" are all acceptable